go.mod

```diff
@@ -12,18 +12,17 @@ require (
 	github.com/go-openapi/jsonreference v0.19.3
 	github.com/go-openapi/spec v0.19.3
 	github.com/go-openapi/swag v0.19.5
-	github.com/golang/protobuf v1.3.3
-	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/golang/protobuf v1.4.2
```

Comment:
@yliaog Can we do that?

Reply:
it is used to marshal document to protobuf, so upgrading to v1.4.2 should be ok.

https://github.com/kubernetes/kube-openapi/blob/1a53ea80ebf47de9717f9a548c26f3d894905964/pkg/handler/handler.go#L188

proto.Marshal(document)